src/App.tsx

```diff
@@ -1,223 +1,68 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
-import { AuthContext } from './components/Auth/AuthContext';
+import React, {
+  // useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+// import { AuthContext } from './components/Auth/AuthContext';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer/Footer';
+import { TodoList } from './components/TodoList/TodoList';
+import { Error } from './components/Errors/Error';
+
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { getTodos } from './api/todos';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<Filter>(Filter.All);
+  const [error, setError] = useState(false);
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  // const user = useContext(AuthContext);
+
+  // useEffect(() => {
+  //   // focus the element with `ref={newTodoField}`
+  //   if (newTodoField.current) {
+  //     newTodoField.current.focus();
+  //   }
+  // }, []);
 
   useEffect(() => {
-    // focus the element with `ref={newTodoField}`
-    if (newTodoField.current) {
-      newTodoField.current.focus();
-    }
+    getTodos(5)
```

Comment:
why do you hardcode arguments for getTodos ?
check what exactly `getTodos` receives and rewrite your code!